windows/clangw/clangw.cpp

```diff
@@ -38,7 +38,8 @@ static option_map _table[] = {{"/nologo", ""},
                               {"/FS", ""},
                               {"-std:c++11", "-std=c++11"},
                               {"-std:c++14", "-std=c++14"},
-                              {"-std:c++17", "-std=c++17"}};
+                              {"-std:c++17", "-std=c++17"},
+                              {"/showIncludes", ""}};
```

Comment:
Was this added to aid in debugging or is it necessary to build?

Reply:
This was added for getting it to build. Ninja builds seem to pass this along to the compiler.